src/test/java/DemoTest.java

```diff
@@ -1,7 +1,31 @@
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import java.util.concurrent.TimeUnit;
 
 public class DemoTest {
 
+  @Test
+  public void incorrectPassAndEmail() {
```

Comment:
your homework "Set up Selenium Framework with Gradle" don't assume implementing first test case. But if you did it I think first test should be a positive like a login with valid credential